aemedge/blocks/promo/promo.css

```diff
@@ -62,43 +80,67 @@
   margin-bottom: var(--udexSpacer8);
 }
 
-.promo div>p a,
-.promo .button-container .button {
-  height: 36px;
-  border-radius: 8px;
-  border: 1px solid;
-  text-align: center;
-  color: var(--udexColorBlue9);
-  border-color: var(--udexColorGrey7);
-  background-color: var(--udexCoreBase);
-  overflow: unset;
-}
-
-
-.promo div>p a,
-.promo .button-container .button.primary {
-  color: var(--udexCoreBase);
-  border-color: var(--udexColorBlue7);
-  background-color: var(--udexColorBlue7);
-  height: auto;
-}
-
-.promo p.button-container {
-  margin: 0;
-  margin-top: var(--udexSpacer24);
-
-}
-
-.promo .button-container {
-  margin: 0;
-}
-
-.promo .button-container a {
-  margin: 0;
-}
+.button-container {
```

Comment:
All styles for '.button-container' and children should go to styles.css directly.

Reply:
This will be covered as part of #268 . There are some other buttons for which rendering logic alos need to be adopted